proto/process/agent.proto

```diff
@@ -102,6 +102,9 @@ message CollectorConnections {
 	map<string, RuntimeCompilationTelemetry> compilationTelemetryByAsset = 21;
 	KernelHeaderFetchResult kernelHeaderFetchResult = 41;
 
+	// CO-RE telemetry
+	map<string, CoReResult> coReTelemetryByAsset = 42;
```

Comment:
Since CO-RE is an acronym, Go naming recommendations suggest we use `CORETelemetryByAsset`